tests/functional/sngtestmod.py

```diff
@@ -39,6 +39,8 @@ def is_open(self):
 
     def send(self, msg):
```

Comment:
Please remove the changenote as well.

Reply:
The NEWS file is already generated and formatted, and I removed this change from it.